### ISSUE TITLE
Suspected errors with qupzilla -reverse

### ISSUE THREAD

**Issue by GreenLunar** [QupZilla/qupzilla#1864]
```
$ qupzilla -reverse
QupZilla: 5 extensions loaded
ToolButton(0x9fbf2e8, name = "navigation-button-back")  does not have a property named  "fallbackIcon"
ToolButton(0x9fd95e8, name = "navigation-button-next")  does not have a property named  "fallbackIcon"
ToolButton(0x9fbf2e8, name = "navigation-button-back")  does not have a property named  "fallbackIcon"
ToolButton(0x9fd95e8, name = "navigation-button-next")  does not have a property named  "fallbackIcon"
```

**Comment by nowrep** [comment]
Which theme are you using?

**Comment by GreenLunar** [comment]
Default style of of Qt and "Linux Default" theme of QupZilla.

This error does not appear when I start QupZilla without `-reverse`.

**Comment by nowrep** [comment]
And your version? I think this is fixed in `master` already.

**Comment by GreenLunar** [comment]
QupZilla 1.8.9

**Comment by nowrep** [comment]
Should be fixed with ae200510645e
